torch/_inductor/autotune_process.py

```diff
@@ -1,10 +1,15 @@
+from __future__ import annotations
```

Comment:
This is to support the type hint `Queue[TuningProcess]` in the 3.8 builds since apparently a subscriptable Queue type was added later.